Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -307,6 +307,12 @@ void ApplicationManagerBase::Rescan()
     GetAssetScanner()->StartScan();
 }
 
+void ApplicationManagerBase::FastScan()
```

Comment:
You might want to remove this for now while there is no UI for it. Not a big deal to keep it besides any code maintenance gotchas, if nothing calls this for a long time, the logic here becomes stale, and someone starts using it in the future, it could have problems.